index.html

```diff
@@ -34,12 +34,12 @@ <h1 class="big-title">
             Food
             <span class="red-span">Hungry</span> delicious.
           </h1>
-          <p>
+          <p class="left-align">
             In order to build a successful brand, every food selector business -
             whether retail, wholesale, needs to make a connection with
             customers.
           </p>
-          <div class="input-button-field">
+          <div class="input-button-field" >
```

Comment:
Use some kind of linter to prevent empty spaces like you have it here after the class declaration.